pilot/pkg/serviceregistry/mock/discovery.go

```diff
@@ -136,10 +136,13 @@ func MakeIP(service *model.Service, version int) string {
 		return ""
 	}
 	ipa, _ := netip.ParseAddr(service.DefaultAddress)
```

Comment:
Prefer check the err

Reply:
thanks for review.
 done it.

Reply:
need some tests as well

Reply:
Oh , sorry , have test it in my local, 
```go test -v controller_test.go controller.go```
Test pass..

Reply:
You forgot to push?

Reply:
Pushd, occur new ci errors..
:grinning::grinning: